parts/linux/cloud-init/artifacts/mig-partition.service

```diff
@@ -5,6 +5,7 @@ Description=Apply MIG configuration on Nvidia A100 GPU
 Restart=on-failure
 ExecStartPre=/usr/bin/nvidia-smi -mig 1
 ExecStart=/bin/bash /opt/azure/containers/mig-partition.sh ${GPU_INSTANCE_PROFILE}
+TimeoutSec=300
```

Comment:
What's the default timeout when this is not added?

Reply:
The default timeout value for normal systemd service is 90s